core/src/test/java/apoc/create/CreateTest.java

```diff
@@ -228,5 +235,66 @@ public void testVirtualFromNodeFunction() throws Exception {
                     assertNull(node.getProperty("born"));
                 });
     }
+    
+    @Test
+    public void testVirtualPath() {
+        db.executeTransactionally("CREATE p=(a:Test {foo: 7})-[:TEST]->(b:Baa:Baz {a:'b'})<-[:TEST_2 {aa:'bb'}]-(:Bar {one:'www'}), \n" +
+                "q=(:Omega {alpha: 'beta'})<-[:TEST_3 {aa:'ccc'}]-(:Bar {one:'jjj'})");
+        testCall(db, "MATCH p=(a:Test {foo: 7})-[:TEST]->(b:Baa:Baz {a:'b'})<-[:TEST_2 {aa:'bb'}]-(:Bar {one:'www'}) WITH p \n" +
+                        "CALL apoc.create.vPath(p) YIELD path RETURN path",
+                (row) -> assertionsFirstVPath((Path) row.get("path")));
+
+        testResult(db, "MATCH p=(a:Test {foo: 7})-[:TEST]->(b:Baa:Baz {a:'b'})<-[:TEST_2 {aa:'bb'}]-(:Bar {one:'www'}), \n" +
+                        "q=(:Omega {alpha: 'beta'})<-[:TEST_3 {aa:'ccc'}]-(:Bar {one:'jjj'}) WITH collect(p)+q as paths \n" +
```

Comment:
```suggestion
                        "q=(:Omega {alpha: 'beta'})<-[:TEST_3 {aa:'ccc'}]-(:Bar {one:'jjj'}) WITH [p, q] as paths \n" +
```
wouldn't this work instead?